pkgs/by-name/op/opentelemetry-collector-builder/package.nix

```diff
@@ -5,17 +5,19 @@
 }:
 buildGoModule rec {
   pname = "ocb";
-  version = "0.112.0";
+  # Also update `pkgs/tools/misc/opentelemetry-collector/releases.nix`
+  # whenever that version changes.
+  version = "0.114.0";
```

Comment:
Is there a way to keep the hashes closer to this version number?

Theres *two* places with hashes in `releases.nix`, pretty far apart and it's probably only a matter of time until some are forgotten to be updated.

Reply:
It might be worth moving this package out of pkgs/by-name and have it close to the rest.

The next bigger work is to write a shared updater script.